Wind-River/cloud-platform-deployment-manager
# Upversion to v2.0.8-3

* Add clock service type for PTP instances (#119)

PTP instances support 4 types of service
('ptp4l', 'phc2sys', 'ts2phc', 'clock'), but the DM
only accepts 3, this commit adds the 'clock' type
to reflect the sysinv configuration.

Test Plan:
1. Create DM configuration with PTP basic conf.
2. Set at least one PTP instance type to 'clock'
3. Apply the DM and verify the ptp instance
- Instance created with service type 'clock'

Signed-off-by: Hugo Brito <hugo.brito@windriver.com>

Signed-off-by: Hugo Brito <hugo.brito@windriver.com>
